controllers/statussync/policy_status_sync.go

```diff
@@ -480,15 +470,25 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, request reconcile.Requ
 				fmt.Sprintf("Policy %s status was updated to %s in cluster namespace %s", hubPlc.GetName(),
 					hubPlc.Status.ComplianceState, hubPlc.GetNamespace()))
 		} else {
-			reqLogger.Info("status match on hub, nothing to update")
+			reqLogger.V(1).Info("status match on hub, nothing to update")
 		}
 	}
 
-	reqLogger.Info("Reconciling complete")
+	reqLogger.V(2).Info("Reconciling complete")
 
 	return reconcile.Result{}, nil
 }
 
+func (r *PolicyReconciler) triggerSpecSyncReconcile(request reconcile.Request) {
+	hubReplicatedPolicy := &unstructured.Unstructured{}
+	hubReplicatedPolicy.SetAPIVersion(policiesv1.GroupVersion.String())
+	hubReplicatedPolicy.SetKind(policiesv1.Kind)
+	hubReplicatedPolicy.SetName(request.Name)
```

Comment:
Is this name correct? Isn't this hubnamespace+.+name? I am confused

Reply:
Yeah, this name is correct because the spec-sync watches the managed cluster namespace on the hub, which is `<namespace>.<name>` and this is the same name in the managed cluster namespace on the managed cluster.

Reply:
request.Name == <namespace>.<name> oh ok!! now